components/screens/post/PostScreen.tsx

```diff
@@ -33,20 +42,23 @@ function PostScreen({
   route: any;
   navigation: NativeStackNavigationProp<any>;
 }) {
+  const [showLoadAll, setShowLoadAll] = useState(true);
+  const [sortType, setSortType] = useState("Top");
+
   const theme = useTheme();
   const post = usePost(
-    route.params && route.params.commentId ? route.params.commentId : null
+    route.params && route.params.commentId ? route.params.commentId : null,
+    sortType as CommentSortType
   );
 
-  const [showLoadAll, setShowLoadAll] = useState(true);
-
   useEffect(() => {
```

Comment:
Like I said above, we'll just get rid of this useEffect and move it to the post hook. Better for us to have one. Not your fault, just a refactor that needed to be done.